EventFilter/GEMRawToDigi/interface/GEBdata.h

```diff
@@ -11,34 +11,47 @@ namespace gem {
   union GEBchamberHeader {
     uint64_t word;
     struct {
-      uint64_t inputStatus : 23;      // Input status (critical) only first 13 bits used
-      uint64_t vfatWordCnt : 12;      // Size of VFAT payload in 64bit words expected to send to AMC13
-      uint64_t inputID : 5;           // GLIB input ID (starting at 0)
-      uint64_t zeroSupWordsCnt : 24;  // Bitmask indicating if certain VFAT blocks have been zero suppressed
+      uint64_t BxmVvV : 11;           // 1st bit BX mismatch VFAT vs VFAT
+      uint64_t BxmAvV : 1;            // BX mismatch AMC vs VFAT
```

Comment:
In the comment of the previous version of the dataformat it was written that "only first 13 bits used". But you are here rewriting the 12th and 13th bit: is the new dataformat still backward compatible as such?

Reply:
The data format is compatible. Before the inputStatus was just lumped together.